floodviz/views.py

```diff
@@ -1,7 +1,41 @@
-from flask import request, render_template, abort
+import json
+import requests
+from flask import render_template
 
 from . import app
+from . import hydrograph_utils
+
 
 @app.route('/home/')
 def home():
-    return render_template('index.html')
\ No newline at end of file
+    # new data store #
+    newd = []
+    sites = app.config['SITE_IDS']
+    start_date = app.config['START_DT']
+    end_date = app.config['END_DT']
+    n_show_series = app.config['N_SERIES']
+    hydro_meta = app.config['HYDRO_META']
+    url_top = app.config['NWIS_SITE_SERVICE_ENDPOINT']
+    sites_value_maxes = {}
+
+    # Set up to retrieve all site ids from url? #
+    for idx, site in enumerate(sites):
```

Comment:
Can the data for all of the sites be retrieved with a single call? I think you can use the sites=x1,x2,x3 parameter. That's preferable to making multiple calls. Parsing may be more difficult but that's preferable to making multiple requests.